Packs/ArcSightESM/Integrations/ArcSightESMv2/ArcSightESMv2_test.py

```diff
@@ -106,3 +105,22 @@ def test_use_rest(mocker, use_rest, cmd_name, expected_rest_endpoint):
         arcsight_cmd()
         last_request = m.last_request
         assert last_request.url == expected_rest_endpoint
+
+
+def test_decode_arcsight_output_event_ids():
```

Comment:
not sure that this UT is relevant, as the problem manifest itself on the server side.

Reply:
I also think it it is only a partial coverage, but I think it might be useful to understand why we cast the ids.
I'll add a more thorough explanation in the test.